prebid-mobile/prebid-mobile-rendering.md

```diff
@@ -0,0 +1,110 @@
+---
+
+layout: page_v2
+title: Rendering Module
+description: Description of how Prebid SDK supports rendering
+sidebarType: 2
+
+---
+
+# Rendering Module Overview
+
+> ⚠️ NOTE: Rendering is an Open Beta Feature
+
+Prebid SDK introduces the rendering facilities as a standalone module. Adding this module won't change anything in your current integration. 
```

Comment:
I would like to edit some of the language for clarity and / spelling mistakes.


Prebid SDK is introducing full ad rendering capabilities into the full suite of product features through an Open Beta as we design, introduce and test various rendering capabilities and interfaces. The Rendering Open Beta features will be available through our Modular structure, leaving intact existing functionality and interface API. All Rendering features will introduce new APIs tied strictly to rendering and will change over time as align the rendering APIs with the existing Prebid SDK APIs.